src/main/resources/assets/bloodmagic/lang/de_DE.lang

```diff
@@ -70,74 +70,150 @@ item.bloodmagic.baseComponent.reagentWater.name=Wasserreagenz
 item.bloodmagic.baseComponent.reagentLava.name=Lavareagenz
 item.bloodmagic.baseComponent.reagentAir.name=Luftreagenz
 item.bloodmagic.baseComponent.reagentFastMiner.name=Reagenz der Eile
-item.bloodmagic.baseComponent.reagentVoid.name=reagenz der Leere
+item.bloodmagic.baseComponent.reagentVoid.name=Reagenz der Leere
 item.bloodmagic.baseComponent.reagentGrowth.name=Wachstumsreagenz
 item.bloodmagic.baseComponent.reagentAffinity.name=Reagenz der elementaren Verbundenheit
 item.bloodmagic.baseComponent.reagentSight.name=Sichtreagenz
 item.bloodmagic.baseComponent.reagentBinding.name=Bindendes Reagenz
-item.bloodmagic.baseComponent.reagentSuppression.name=Unterdrückendes Reagenz
+item.bloodmagic.baseComponent.reagentSuppression.name=Verdrängendes Reagenz
+item.bloodmagic.baseComponent.reagent_blood_light.name=Blutlampenreagenz
+item.bloodmagic.baseComponent.reagent_magnetism.name=Magnetismusreagenz
+item.bloodmagic.baseComponent.reagent_haste.name=Reagenz der Hast
+item.bloodmagic.baseComponent.reagent_bridge.name=Reagenz der Phantombrücke
+item.bloodmagic.baseComponent.reagent_compression.name=Kompressionsreagenz
+item.bloodmagic.baseComponent.reagent_severance.name=Trennungsreagenz
+item.bloodmagic.baseComponent.reagent_holding.name=Aufbewahrungsreagenz
+item.bloodmagic.baseComponent.reagent_claw.name=Klauenreagenz
+item.bloodmagic.baseComponent.reagent_bounce.name=Elastizitätsreagenz
+item.bloodmagic.baseComponent.reagent_frost.name=Frostreagenz
+item.bloodmagic.baseComponent.reagent_teleposition.name=Telepositionsreagenz
+item.bloodmagic.baseComponent.reagent_transposition.name=Transpositionsreagenz
+
+item.bloodmagic.baseComponent.frame_part.name=Rahmenteile
+item.bloodmagic.baseComponent.sand_iron.name=Eisensand
+item.bloodmagic.baseComponent.sand_gold.name=Goldsand
+item.bloodmagic.baseComponent.sand_coal.name=Kohlesand
+item.bloodmagic.baseComponent.plant_oil.name=Pflanzenöl
+item.bloodmagic.baseComponent.sulfur.name=Schwefel
+item.bloodmagic.baseComponent.saltpeter.name=Saltpeter
+item.bloodmagic.baseComponent.neuro_toxin.name=Nervengift
+item.bloodmagic.baseComponent.antiseptic.name=Antiseptikum
+item.bloodmagic.baseComponent.catalyst_length_1.name=Einfacher Verlängerungskatalysator
+item.bloodmagic.baseComponent.catalyst_power_1.name=Einfacher Machtkatalysator
+
+item.bloodmagic.cutting_fluid.basic.name=Einfache Schneidelösung
+item.bloodmagic.cutting_fluid.explosive.name=Explosives Pulver
+
+item.bloodmagic.demonCrystal.default.name=Kristall dämonischen Willens
```

Comment:
"Willens" is a bad translation. Think something more like willpower and volition

Reply:
Do you just want to change it for the crystals, everything base (i.e. Crystal of demonic Willpower, Raw/Corrosive/Destructive/Vengeful/Steadfast Crystal of demonic Willpower, Demonic Willpower, Raw/Corrosive/Destructive/Vengeful/Steadfast demonic Willpower)?

Reply:
"Willens" is a special grammar case of "Wille" both refer to the will or the "I want this" of a Person. "Willenskraft" would be the proper translation for "Willpower", however nobody would use it in such an extensive context. It would be too long and too repetitive.

Reply:
Although it does have that meaning, it's a bit obscure. I think it's just easier to translate it to "Will" as a borrowed noun.

Reply:
Leaving it untranslated would make it even more obscure because people won't expect an untranslated word there, especially if it has meaning in German.

Reply:
Sounds good.